graph_annotation.go

```diff
@@ -0,0 +1,18 @@
+package mackerel
+
+// GraphAnnotation parameters to post graph annotation
+type GraphAnnotation struct {
+	Service     string   `json:"service,omitempty"`
+	Roles       []string `json:"roles,omitempty"`
+	From        int64    `json:"from,omitempty"`
```

Comment:
Though using `*time.Time` looks nice, but already existing codes in mackerel-client-go uses int64 for timestamp-like-integers.  If we change, maybe we should change all of them?